src/components/PathwayExercise/index.js

```diff
@@ -212,19 +219,25 @@ function PathwayExercise() {
                   onClick={nextClickHandler}
                 />
               </Toolbar>
-              <Select
-                IconComponent={() => null}
-                disableUnderline
-                value={language}
-                onChange={(e) => {
-                  setLanguage(e.target.value);
-                }}
-                variant="standard"
-              >
-                <MenuItem value="en">English</MenuItem>
-                <MenuItem value="hi">Hindi</MenuItem>
-                <MenuItem value="mr">Marathi</MenuItem>
-              </Select>
+              {availableLang?.length === 1 ? (
+                <MenuItem value={availableLang[0]}>
+                  {Lang[availableLang[0]]}
+                </MenuItem>
+              ) : (
+                <Select
+                  disableUnderline
+                  value={language}
+                  IconComponent={() => null}
+                  onChange={(e) => {
+                    setLanguage(e.target.value);
+                  }}
+                  variant="standard"
+                >
+                  {availableLang.map((lang) => {
+                    return <MenuItem value={lang}>{Lang[lang]}</MenuItem>;
+                  })}
+                </Select>
+              )}
```

Comment:
I recommend using a constant for the language menu so you can avoid duplication (Whether it's 1 language or multiple ones, the following should work):
```
const langMenu = availableLang.map((lang) => <MenuItem value={lang}>{Lang[lang]}</MenuItem>);
```
Also, to avoid additional code duplication, I recommend you use a component here, so it'd look something like this:
```
function languageSelectMenu() {
  const langMenu = availableLang.map((lang) =><MenuItem value={lang}>{Lang[lang]}</MenuItem>);
  return (
    availableLang.length === 1 ? 
      langMenu :
      <Select
        disableUnderline
        value={language}
        IconComponent={() => null}
        onChange={(e) => {
          setLanguage(e.target.value);
        }}
        variant="standard"
      >
        langMenu
      </Select>
  )
}
```